app/lib/support/support_page_controller.dart

```diff
@@ -42,11 +61,66 @@ class SupportPageController extends ChangeNotifier {
     });
   }
 
+  /// Returns the unencoded URL for the video call appointments page, with
+  /// prefilled user ID, user name, and user email (for non-private Apple
+  /// email).
+  ///
+  /// Example URL:
+  /// ```
+  /// https://sharezone.net/sharezone-plus-video-call-support?userId=userId123&name=My Cool Name&email=my@email.com
+  /// ```
+  /// Note: The URL returned by this method should be encoded before usage.
```

Comment:
Why is the URL unencoded? You don't explicitly encode it afterwards, is it done by `launchURL(url, context: context);`? This should be clarified in the documentation.

Reply:
Yes, it's done by the `launchURL` method

Reply:
I added "Note: The URL is not encoded because the `launchURL` method will encode the URL."